src/client/Client.cc

```diff
@@ -9623,6 +9628,7 @@ Inode *Client::open_snapdir(Inode *diri)
     in->ctime = diri->ctime;
     in->btime = diri->btime;
     in->size = diri->size;
+    in->change_attr = diri->change_attr;
```

Comment:
You appear to have removed this, and I didn't mean to say it was *wrong*, I just wanted to double-check. :)
(And actually we do allow snaps to be renamed at present, which could maybe bump the change attr...not sure how that will play out in future or if it's a thing we want to support).